### ISSUE TITLE
Failed to load plugin vaughnlive

### ISSUE THREAD

**Issue by karlo2105** [beardypig/streamlink-portable#11]
After having updated streamlink portable nightlies build I get such message in command line : 

```
Failed to load plugin vaughnlive:
  File "imp.py", line 234, in load_module
  File "imp.py", line 172, in load_source
  File "<frozen importlib._bootstrap>", line 693, in _load
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 673, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "C:\Users\Ddr\Downloads\Temp\streamlink\packages\streamlink\plugins\vaugh
nlive.py", line 5, in <module>
    import websocket
ImportError: No module named 'websocket'

usage: streamlink-script.py [OPTIONS] <URL> [STREAM]

Use -h/--help to see the available options or read the manual at https://streaml
ink.github.io
```

With your latest commit it's same result.

**Comment by beardypig** [comment]
This should be fixed now, I just pushed out `0.8.1`.

**Comment by karlo2105** [comment]
Thank you it's fine now. ;)
